go-controller/pkg/ovn/master.go

```diff
@@ -639,12 +629,8 @@ func (oc *Controller) syncGatewayLogicalNetwork(node *kapi.Node, l3GatewayConfig
 
 	// Add cluster load balancers to GR for Host -> Cluster IP Service traffic
 	if config.Gateway.Mode != config.GatewayModeLocal {
-		clusterLBs := []string{oc.TCPLoadBalancerUUID, oc.UDPLoadBalancerUUID}
-		if oc.SCTPSupport {
-			clusterLBs = append(clusterLBs, oc.SCTPLoadBalancerUUID)
-		}
 		gr := util.GetGatewayRouterFromNode(node.Name)
-		for _, clusterLB := range clusterLBs {
+		for _, clusterLB := range oc.loadbalancersUUID {
```

Comment:
i think the new name is more confusing than the old name. maybe oc.clusterLBsUUIDs would be better? The new name makes me thin all the load balancer uuids are being stored here, but really its just the cluster load balancers right?